browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -533,3 +570,51 @@ IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
   EXPECT_EQ("third-party-a.com", third_party_values.session_storage);
   EXPECT_EQ("name=third-party-a.com", third_party_values.cookies);
 }
+
+class EphemeralStorageKeepAliveDisabledBrowserTest
+    : public EphemeralStorageBrowserTest {
+ public:
+  EphemeralStorageKeepAliveDisabledBrowserTest()
+      : EphemeralStorageBrowserTest() {
+    scoped_feature_list_.InitAndDisableFeature(
```

Comment:
`scoped_feature_list_` should be in this class.

Reply:
it was previously used in the base class, but you are right it doesn't need to be there anymore